product_pack/tests/test_product_pack.py

```diff
@@ -75,7 +75,7 @@ def test_get_pack_lst_price(self):
         component_2.product_id.list_price = 15.0
         component_3 = self.env.ref("product_pack.pack_cpu_detailed_components_4")
         component_3.product_id.list_price = 5.0
-        self.assertEqual(50.0, pack.lst_price)
+        self.assertEqual(80.75, pack.lst_price)
```

Comment:
Why this change?

Reply:
This is because we are fixing the way the pack “totalized in main product” price is computed. According to the [README](https://github.com/OCA/product-pack/blob/3e8eb93b8431b5808aab2174b21ae357e70145f0/product_pack/README.rst?plain=1#L66-L68): the pack totalized price “will be its price plus the sum of all the components prices” but in the 16.0 the price is only the sum of the components. 
In the test:

- Sum of the components: 30+15+5 = 50.0
- Product Pack price: 30.75

Then the result of the test should be 50.0 + 30.75 = 80.75